src/FluentCMS.Web.UI/Components/Core/Form/Input.razor

```diff
@@ -6,8 +6,8 @@
         <div class=@this.ClassName("InputIconWrapper")>
             <Icon Name="@IconName" />
         </div>
-        <input 
-            type="text" 
+        <input
+            type=@this.Type.ToString().Replace("_", "-")
```

Comment:
We have a `FromPascalCaseToKebabCase()` extension method for this logic. can we use that?